### ISSUE TITLE
opens sessions, posts nothing and does not respond to commands

### ISSUE THREAD

**Issue by NFS11** [moom825/Discord-RAT-2.0#103]
literally makes a  sessions, then does nothing else

**Comment by MRX8001** [comment]
> literally makes a  sessions, then does nothing else

I suspect that there has been an update with Discord, it was very strange that it was working a few days ago and now it doesn't work anymore, it opens a session but the commands don't respond

**Comment by Lbozoisback** [comment]
It cant be the new update because the bot still makes the sessions but without @here in it

**Comment by Itzkristis** [comment]
> It cant be the new update because the bot still makes the sessions but without @here in it

No it dosent atleast for me

**Comment by socnuz** [comment]
@here  NOT working now :(

**Comment by NFS11** [comment]
how did you lot get it working? i did everything. it opens new channel and session, but says nothing and wont respond to commands. i authorised it, made it a bot, put it in my server, gave it administrator privaliges and it still does nothing. it even shows as online. is there anything im missing? i gave it token and guild id. i can send screenshots too

**Comment by NFS11** [comment]
![Screenshot 2024-07-27 190740](https://github.com/user-attachments/assets/19c04099-9e9d-4715-8d9b-7e9164984829)
here is when i opened client built and it made a session and posted nothing

**Comment by NFS11** [comment]
![Screenshot 2024-07-27 191025](https://github.com/user-attachments/assets/55e5c02f-d721-47e4-b1cd-e4ec8759d484)
could also be because of this? i cant really change those settings for some reason

**Comment by MootjeD** [comment]
same issue

**Comment by NFS11** [comment]
hi all,

i am unsure if this is the reason but: i may have disabled message intents so maybe thats what happened? i will try again on an alt account and see what happens.

**Comment by NFS11** [comment]
yeah you lot i forgot the message intents. thats why. it works now.
